bin/http-server

```diff
@@ -92,10 +92,16 @@ else {
   listen(port);
 }
 
+/* jshint evil:true */
 function listen(port) {
+  var cache = argv.c;
+  try {
+    cache = eval(argv.c);
+  }
+  catch (e) {}
```

Comment:
If we get into the [empty] catch block, what will cache be set to? Are we certain it will still be equal to `argv.c`? Should we use an intermediate variable?

Reply:
Added intermediate variable.